pkg/sfu/buffer/videolayerutils.go

```diff
@@ -104,13 +104,13 @@ func RidToSpatialLayer(rid string, trackInfo *livekit.TrackInfo, ridSpace VideoL
 			return 2
 
 		case lp[livekit.VideoQuality_LOW] && lp[livekit.VideoQuality_MEDIUM]:
-			logger.Warnw("unexpected rid f with only two qualities, low and medium", nil, "trackID", trackInfo.Sid, "trackInfo", logger.Proto(trackInfo))
+			logger.Warnw("unexpected rid with only two qualities, low and medium", nil, "trackID", trackInfo.Sid, "trackInfo", logger.Proto(trackInfo), "rid", ridSpace[2])
```

Comment:
are these logs all supposed to be the same index?

Reply:
Yeah, they are all asking for `f` (highest layer or third layer) when less than three layers are specified in trackInfo. With dynamic rids, they are occupying the last slot in the rid array.

Reply:
just wanted to be sure it wasn't an out of range error